src/utilcode/prettyprintsig.cpp

```diff
@@ -686,10 +686,6 @@ static HRESULT PrettyPrintTypeA(
         str = "class "; 
         goto DO_CLASS;
         
-    case ELEMENT_TYPE_CMOD_REQD:
-        str = "required_modifier ";
-        goto DO_CLASS;
-        
     case ELEMENT_TYPE_CMOD_OPT:
```

Comment:
`ELEMENT_TYPE_CMOD_OPT` should be on the same plan as `ELEMENT_TYPE_CMOD_REQD`

Reply:
My reasoning was to keep the cases that call DO_CLASS together, but moved it back now.